konflux_build_args.conf

```diff
@@ -0,0 +1,4 @@
+BUILDER_IMAGE=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.2
+RUNTIME_IMAGE=registry.redhat.io/rhel9-4-els/rhel-minimal:9.4-149
+YQ_IMAGE=quay.io/konflux-ci/yq:latest@sha256:5ff4dd745c6f4cc67ae4f00fd2a38dd31f7d99c95dd7ad4476d6a6307a0f40a0
```

Comment:
the YQ_IMAGE (and the rest of the arguments here for that matter) get updated automatically by mintmaker?

Reply:
ok ,  I can see this is set in renovate.json